File: sdkwork-app-sdk-typescript/src/types/share-visit-form.ts
/** 分享访问请求 */
export interface ShareVisitForm {
  /** 分享ID */
  shareId?: string;
  /** 访问类型 */
  visitType?: string;
  /** 用户代理 */
  userAgent?: string;
  /** IP地址 */
  ipAddress?: string;
  /** 设备信息 */
  deviceInfo?: string;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/share_visit_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ShareVisitForm:
    """分享访问请求"""
    share_id: str = None
    visit_type: str = None
    user_agent: str = None
    ip_address: str = None
    device_info: str = None
